Assets/MixedRealityToolkit.Services/BoundarySystem/XR2018/MixedRealityBoundarySystem.cs

```diff
@@ -78,40 +58,37 @@ protected override List<Vector3> GetBoundaryGeometry()
             return boundaryGeometry;
         }
 
-        /// <summary>
-        /// Updates the <see href="https://docs.unity3d.com/ScriptReference/XR.TrackingSpaceType.html">TrackingSpaceType</see> on the XR device.
-        /// </summary>
-        protected override void SetTrackingSpace()
-        {
-            TrackingSpaceType trackingSpace;
+        #region Obsolete
 
-            // In current versions of Unity, there are two types of tracking spaces. For boundaries, if the scale
-            // is not Room or Standing, it currently maps to TrackingSpaceType.Stationary.
-            switch (Scale)
-            {
-                case ExperienceScale.Standing:
-                case ExperienceScale.Room:
-                    trackingSpace = TrackingSpaceType.RoomScale;
-                    break;
-
-                case ExperienceScale.OrientationOnly:
-                case ExperienceScale.Seated:
-                case ExperienceScale.World:
-                    trackingSpace = TrackingSpaceType.Stationary;
-                    break;
-
-                default:
-                    trackingSpace = TrackingSpaceType.Stationary;
-                    Debug.LogWarning("Unknown / unsupported ExperienceScale. Defaulting to Stationary tracking space.");
-                    break;
-            }
+        /// <inheritdoc/>
+        [System.Obsolete("Use MixedRealityToolkit.AppScale instead")]
+        protected override void SetTrackingSpace() { }
```

Comment:
Please do not remove the implementation of this method when marking obsolete. Existing code should continue to function.

Reply:
If possible, it is better to re-implement obsolete methods using the new code.